scripts/liftoff.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
```

Comment:
Best practice to always have `set -eu` at top to ensure errors stop the process (e) and undefined variables throw (u).